packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -169,10 +170,11 @@ watch(
 			// Start 2nd simulation to get sample simulation from dill
 			const newInterventionResponse = await createInterventionPolicyFromOptimize(modelConfigId.value as string, optId);
 
-			const preForecastResponce = await startForecast(undefined);
-			const preForecastId = preForecastResponce.id;
-			const postForecastResponce = await startForecast(newInterventionResponse);
-			const postForecastId = postForecastResponce.id;
+			const preForecastResponse = startForecast(undefined);
+			const postForecastResponse = startForecast(newInterventionResponse);
+			const forescastResults = await Promise.all([preForecastResponse, postForecastResponse]);
+			const preForecastId = forescastResults[0].id;
+			const postForecastId = forescastResults[1].id;
```

Comment:
Not necessary, just FYI

```suggestion
			const [{ id: preForecastId }, { id: postForecastId }] = forecastResults;
```

Reply:
Thank you for this!